packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -62,7 +65,7 @@ export const getStyles = memoizeFunction((
       borderWidth: '1px',
       borderStyle: 'solid',
       borderColor: checkboxBorderColor,
-      marginRight: '8px',
+      margin: '0 4px',

```

Comment:
Question : Is there some code that auto RTL fies styles for us ? I think that was the reasoning behind this being changed from margin: '0 4px' to  marginRight.